modules/installation-dns-user-infra.adoc

```diff
@@ -12,6 +12,11 @@
 [id="installation-dns-user-infra_{context}"]
 = User-provisioned DNS requirements
 
+DNS is used for name resolution (A/AAAA or CNAME records) and reverse name resolution (PTR records).
+The reverse records are important because {op-system} uses the reverse records to set the host name for
```

Comment:
s/{op-system}/{op-system-first} 

Since this is the first reference.

Reply:
@bmcelvee, is -first used in the first reference in any file?  I figured it would be used in the first instance in the assembly. It doesn't happen to be used there either, in this case.  My instinct is to add the information about when to use -first to the doc guidelines.

Reply:
We've been using it for the first reference in every file because with reuse, we don't what order every file will appear at any given time. (It does read a bit odd within the assembly.)

Reply:
Just one thing to swap around, then we'll be good to merge! 

s/{op-system}/{op-system-first}
s/{product-title-first}/{product-title}